src/lib.rs

```diff
@@ -1062,6 +1077,42 @@ where
         self.oauth2_client.exchange_code(code)
     }
 
+    ///
+    /// Creates a request builder for exchanging a device code for an access token.
+    ///
+    /// See https://tools.ietf.org/html/rfc8628#section-3.4
```

Comment:
let's linkify this URL by wrapping it in `<...>`. I think `cargo doc` warns when URLs don't have this. some of the other rustdocs in this crate have the same issue (not sure if rustdoc had that feature back then or if I was just unaware of it), but let's avoid new links that trigger warnings.